.env

```diff
@@ -4,3 +4,7 @@ POSTGRES_HOST=resources-postgres
 POSTGRES_DB=resources-psql
 FLASK_ENV=development
 FLASK_DEBUG=1
+SEARCH_USER=search_user
+SEARCH_KEY=search_key
+SEARCH_RESOURCE_INDEX=search_resource_index
+INDEX_SEARCHED=FALSE
```

Comment:
Should this be `INDEX_CHECKED` instead of `INDEX_SEARCHED`? This doesn't appear to be used anywhere, and `INDEX_CHECKED` doesn't seem to be declared anywhere even though its used

Reply:
Yeah, that was a typo. Fixed in next commit